content/en/providers/check-in/sp/_index.md

```diff
@@ -1090,7 +1136,9 @@ The following information about the authenticated user can be provided by EGI
 Check-in in order to control user access to resources:
 
 1. VO/group membership and role information about the authenticated user
+1. Capabilities
 1. Level of Assurance (LoA)
```

Comment:
Level of Assurance (LoA) --> Identity Assurance

Reply:
Done